diff --git a/esm/utils.js b/esm/utils.js
index 946aec183a3a0115cc3ca75b1d119d4741a354b7..3ffe2c8354e6ff3124de1eb9c03a5711bc91c983 100644
--- a/esm/utils.js
+++ b/esm/utils.js
@@ -4,7 +4,6 @@
 // rewrites import from `crypto` to `cryptoNode`, which imports native module.
 // Makes the utils un-importable in browsers without a bundler.
 // Once node.js 18 is deprecated, we can just drop the import.
-import { crypto } from '@noble/hashes/crypto';
 // Cast array to different type
 export const u8 = (arr) => new Uint8Array(arr.buffer, arr.byteOffset, arr.byteLength);
 export const u32 = (arr) => new Uint32Array(arr.buffer, arr.byteOffset, Math.floor(arr.byteLength / 4));
